packages/bento-design-system/src/Table/cells.tsx

```diff
@@ -85,17 +86,17 @@ export function LabelCell({ value, column: { align } }: CellProps<{}, LocalizedS
   );
 }
 
-export function createLinkCell(Link: FunctionComponent<LinkProps>) {
-  return function LinkCell({
-    value,
-    column: { align },
-  }: CellProps<{}, { href: string; label: LocalizedString; isDisabled?: boolean }>) {
-    return (
-      <Box padding={16} textAlign={align}>
-        <Link {...value} />
-      </Box>
-    );
-  };
+export function LinkCell({
+  value: { label, href, isDisabled },
+  column: { align },
+}: CellProps<{}, { href: string; label: LocalizedString; isDisabled?: boolean }>) {
+  return (
+    <Box padding={16} textAlign={align}>
+      <Label size="large">
+        {makeTextChildrenFromElements(link(label, { href, disabled: isDisabled }))}
+      </Label>
+    </Box>
+  );
 }
```

Comment:
Is it intended that `LinkCell` only supports real links and not link buttons?

Reply:
we already have an `IconButtonCell`. I don't think it makes a lot of sense to have a linkButton here